tests/kickstart_tests/escrow-cert.ks

```diff
@@ -0,0 +1,107 @@
+%pre
+# Create an nss database for the escrow certifcate
+mkdir -p /tmp/escrow_test/nss
+certutil -d /tmp/escrow_test/nss --empty-password -N
```

Comment:
> Does the database need to be created for the escrow to work or is it only needed for the test to verify the results? IOW, do we require people using escrow having this in their kickstarts?

Normally, the NSS database would be on a separate system, and the public certificate would be available over http. This test needs access to the database in order to decrypt the escrow packets to make sure they work, though.